e2e/run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Run Manticore's e2e tests. See the README.md for more details.
+
+# First, build the e2e binary itself, which acts as the server.
+export MANTICORE_E2E_TARGET_BINARY="$(
+  cargo b -p e2e --message-format=json \
+    | tr '\n' ' ' \
+    | perl -pe 's/^.+"executable":"(.+?)".+$/$1/g'
```

Comment:
As much as I'd love to complain about perl, the python solution here is also ugly:

```
python -c "import json,sys;print([x for x in [json.loads(line).get('executable') for line in sys.stdin] if x is not None][0])"

## or

python -c "import json,sys;print(list(filter(bool,[json.loads(line).get('executable') for line in sys.stdin]))[0])"
```

There's probably some nice solution with `jq`, but I don't really know that tool, nor is it ubiquitous.

I'm not suggesting a change, just ranting that perl will never go away.

Reply:
Be happy it's not sed.